sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -56,11 +56,18 @@ static void _check_netdev_capabilities(netdev_t *dev);
 static void *_gnrc_netif_thread(void *args);
 static void _event_cb(netdev_t *dev, netdev_event_t event);
 
+typedef struct {
+    gnrc_netif_t *netif;
+    mutex_t init_done;
+    int result;
+} netif_ctx_t;
```

Comment:
As this is an internal type, maybe mark it as such

```suggestion
} _netif_ctx_t;
```